components/footer.tsx

```diff
@@ -1,3 +1,50 @@
-export default function Footer() {
-  return <footer className="bg-light px-4 py-8 md:p-16">footer</footer>
+/* eslint-disable @next/next/no-img-element */
```

Comment:
it's better if you create an .eslintrc file with a rule to mute that warning instead of putting it in every file.